azure-pipelines/testrun.runsettings

```diff
@@ -6,11 +6,19 @@
           <CodeCoverage>
             <ModulePaths>
               <Include>
-                <ModulePath>Microsoft\.SqlTools.*\.dll$</ModulePath>
-                <ModulePath>Microsoft\.SqlTools.*\.exe$</ModulePath>
+                <ModulePath>.*SqlTools.*</ModulePath>

```

Comment:
not sure I understand this change. We dont want to include dlls/exes from child directories?

Reply:
It seems the covrun tools use the full module path, not just the name, so each entry has to start with ".*". The "dll" and "exe" suffixes are not really needed since the tool only looks at those kinds of modules anyway.
The heuristic "any dll whose name contains SqlTools" seems like a reasonable starting point and if we find noise in the data we can tweak the filter later.